src/app/project/controllers/collect-records.ctrl.js

```diff
@@ -354,38 +369,66 @@ angular.module('app.project').controller('CollectRecordsCtrl', [
             utils.template(utils.messages.deleteRecordWarning, args)
           );
         },
-        filterMethod: function(showMethod, protocol) {
+        filterMethod: function(item) {
+          var selected = item.selected;
```

Comment:
Again not seeing need for additional vars

Reply:
Agreed